apps/src/templates/instructions/InlineFeedback.jsx

```diff
@@ -7,11 +7,8 @@ const InlineFeedback = ({ extra, message, styles, borderColor }) => {
   // easily detect its presence. This class should NOT be used for
   // styling.
   return (
-    <ChatBubble
-      borderColor={borderColor}
-      className="uitest-topInstructions-inline-feedback"
-    >
-      <p>{message}</p>
+    <ChatBubble borderColor={borderColor}>
+      <p className="uitest-topInstructions-inline-feedback">{message}</p>
```

Comment:
probably, but I'm unsure what it would be, and equally unsure that the advantage gained by finding the better way would be work the research time it would take to find it.